scipy/special/add_newdocs.py

```diff
@@ -627,12 +672,40 @@ def add_newdoc(place, name, doc):
     E : ndarray
         Value of the elliptic integral.
 
+    Notes
+    -----
+    Wrapper for the Cephes [1]_ routine `ellpe`.
+
+    For `m > 0` the computation uses the approximation,
+
+    .. math:: E(m) \\approx P(1-m) - (1-m) \\log(1-m) Q(1-m),
+
+    where :math:`P` and :math:`Q` are tenth-order polynomials.  For
+    `m < 0`, the relation
+
+    .. math:: E(m) = E(m/(m - 1)) \\sqrt(1-m)
+
+    is used.
+
+    Accuracy:
+
+    ==========   ======     ========     =======     =======
+    arithmetic   domain     # trials      peak         rms
+    ==========   ======     ========     =======     =======
+       IEEE       0, 1       10000       2.1e-16     7.3e-17
+    ==========   ======     ========     =======     =======
```

Comment:
I think this and similar tables should be omitted from the python-level docstrings: any benchmark results need to be reproducible, and these are not. At least I've no idea how to reproduce these numbers, do you? And unless I'm missing something obvious, I think the effort replicating these is better spent elsewhere (for instance: moving more information from C/Fortran comments to python docstrings).